src/state.ts

```diff
@@ -15,17 +22,18 @@ export async function refreshPiggybankState(rpc: ConcordiumGRPCClient, contract:
     console.debug(`Refreshing piggybank state for contract ${contract.index.toString()}`);
     const { version, name, index, methods } = contract;
 
-    const expectedMethods = ['insert', 'smash', 'view'].map((m) => `${name}.${m}`);
-    if (!expectedMethods.every(methods.includes.bind(methods))) {
+    const expectedMethods = ['insert', 'smash', 'view'].map((m) => `${ContractName.toString(name)}.${m}`);
+    if (!expectedMethods.every((e) => methods.map(ReceiveName.toString).includes(e))) {
```

Comment:
```suggestion
    const methodNames = methods.map(ReceiveName.toString);
    if (!expectedMethods.every((e) => methodNames.includes(e))) {
```

Sux that there isn't a `includesAll`...